src/main/java/lotto/controller/AmountInput.java

```diff
@@ -0,0 +1,24 @@
+package lotto.controller;
+
+import camp.nextstep.edu.missionutils.Console;
+import lotto.model.Constants;
+import lotto.model.Lotto;
+
+public class AmountInput {
+    private final int amount;
+
+    public AmountInput() {
+        try {
+            System.out.println("구입금액을 입력해 주세요.");
+            amount = Integer.parseInt(Console.readLine().trim());
+            Lotto.checkAmount(amount);
+            System.out.println();
+        } catch (NumberFormatException e) {
+            throw new IllegalArgumentException(Constants.INVALID_AMOUNT);
+        }
+    }
+
+    public int getAmount() {
+        return amount;
+    }
```

Comment:
조금 어려워하셔서 예시를 들어봐서 하나의 메서드로 바꿔볼게요!
```suggestion
public class AmountInput {

    public int inputAmount() {
          try {
                  System.out.println("구입금액을 입력해 주세요.");
                  amount = Integer.parseInt(Console.readLine().trim());
                  Lotto.checkAmount(amount);
                  System.out.println();
                  return amount;
              } catch (NumberFormatException e) {
                  throw new IllegalArgumentException(Constants.INVALID_AMOUNT);
            }
    }
```